pkg/gitproviders/repo_url.go

```diff
@@ -87,23 +87,11 @@ func (n RepoURL) Protocol() RepositoryURLProtocol {
 
 func getOwnerFromURL(url url.URL, providerName GitProviderName) (string, error) {
 	url.Path = strings.TrimPrefix(url.Path, "/")
-
 	parts := strings.Split(url.Path, "/")
 	if len(parts) < 2 {
 		return "", fmt.Errorf("could not get owner from url %v", url.String())
 	}
-
-	if providerName == GitProviderGitLab {
-		if len(parts) > 3 {
-			return "", fmt.Errorf("a subgroup in a subgroup is not currently supported")
-		}
-
-		if len(parts) > 2 {
-			return parts[0] + "/" + parts[1], nil
-		}
-	}
-
-	return parts[0], nil
+	return strings.Join(parts[:len(parts)-1], "/"), nil
```

Comment:
could someone provide a url with `len(parts)>4`? in that case it shouldn't be valid right?

Reply:
Great question, I'm not sure what the valid max depth of subgroups are. I've tested at 3, we should be able to handle however many gitlab can, which doesn't seem to be documented afaict.

If there is some limit then an error will appear when we attempt to create the PR, the user can fix their URL at that point I guess.

Reply:
apparently up to 20 subgroups can be created according to [gitlab](https://about.gitlab.com/solutions/subgroups/#:~:text=You%20can%20create%20up%20to,an%20incredible%20level%20of%20flexibility.)